### ISSUE TITLE
Product prices are not accessible to screen readers

### ISSUE THREAD

**Issue by MichaelHeinzman** [adobe/storefront-product-listing-page#44]
We received an accessibility audit that stated the product price needed to notify screen readers that it is a discounted or original price.
